pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
```

Comment:
@krancour actually I take that back. This isn't intended to test the in memory implementation of APIClient but to test apiclient_util, which is in the apiclient package.

That being said, moving this to the mem package would still work but doesn't seem like the right thing to do..

That's why I moved both apiclient_util and the test to its own util package, but someone made the suggestion to leave that here and instead make a new test package instead.